resources/data/elephpants.json

```diff
@@ -591,6 +591,14 @@
             "sponsor": "Laravel Community",
             "year": 2023,
             "image": "74-aubrey.jpg"
+        },
+        {
+            "id": 75,
+            "name": "Dyflexis",
+            "description": "Dyflexis",
```

Comment:
@dseguy maybe you can share something about the elephpant, because I had to go and you did not finish talking about the company 🙂.

Reply:
These "descriptions" aren't really descriptions btw, some of them just copy the Elephpant name. 

Was this Elephpant released for a specific reason? E.g. an event? For example the Laravel ones are released for Laracon events, so that's the description for them. If this Elephpant was released for a specific reason, we can use that as the description!

Reply:
The backstory should go on https://github.com/philipsharp/afieldguidetoelephpants and not the description field

Reply:
The elephpant name is 'Flexy' and the year is actually 2023. 
Nothing else to add. The picture is fine.

Reply:
```suggestion
            "description": "Flexy",
```